runtime/pom.xml

```diff
@@ -56,6 +56,11 @@
             <artifactId>format</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.codepoetics</groupId>
```

Comment:
Please don't use no-name libraries. If you need a good library which helps you with using stream nicely (and not only) you could use https://github.com/vavr-io/vavr it's production ready and very solid

Reply:
I use a Guava stream function to solve the problem and I remove this library from pom.xml.